src/test/java/com/epam/tc/gdi/hw1/tests/TestsInit.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tc.gdi.hw1.tests;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import com.epam.tc.gdi.hw1.site.pages.JdiTestSite;
+import com.epam.tc.gdi.hw1.tests.steps.Preconditions;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public class TestsInit {
+
+    @BeforeSuite
+    void setUp() {
+        PageFactory.initElements(JdiTestSite.class);
+        JdiTestSite.homePage.open();
+    }
+
+    @BeforeMethod
+    void loginIfNot() {
+        Preconditions.loginIfNot();
+    }
+
+    @AfterTest
+    void tearDown() {
+        WebDriverUtils.killAllSeleniumDrivers();
```

Comment:
It's not required to kill browser after each test, it's enough to close all after suite

Reply:
Done